atlasdb-client/src/main/java/com/palantir/atlasdb/stream/BlockGetter.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.stream;
+
+import java.io.OutputStream;
+
+interface BlockGetter {
+    void get(Integer firstBlock, Integer numBlocks, OutputStream destination);
```

Comment:
They take boxed primitives as a hangover from how the code used to look like - it was a templated `BiConsumer<Integer, OutputStream>`, because you can't do `BiConsumer<int, OutputStream>`. Now that we have our own interface, I can change this back.